src/main/resources/content/k8s/ensure-containers-are-immutable-.json

```diff
@@ -1 +1,8 @@
-{"severity":"medium","rule":"$.spec.template.spec.containers[*].securityContext.readOnlyRootFilesystem exists and  is false","id":"c448b01c-7f95-4e9f-97e1-c640733af44f","enabled":true,"resourceType":"k8s","policy":"ensure containers are immutable "}
\ No newline at end of file
+{
+  "severity": "medium",
+  "rule": "$.spec.template.spec.containers[*].securityContext.readOnlyRootFilesystem exists and  is false",
+  "id": "c448b01c-7f95-4e9f-97e1-c640733af44f",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "ensure containers are immutable "
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.